azure-pipelines.yml

```diff
@@ -108,19 +108,20 @@ jobs:
           codeCoverageTool: Cobertura
           summaryFileLocation: '$(System.DefaultWorkingDirectory)/qcodes/coverage.xml'
       - script: |
-          CALL C:\Miniconda\condabin\conda_hook.bat
-          CALL conda activate qcodes
-          cd docs
-          REM Turn warnings into errors
-          set SPHINXOPTS=-W -v
+          CALL C:\Miniconda\condabin\conda_hook.bat && ^
+          CALL conda activate qcodes && ^
+          cd docs && ^
+          REM Turn warnings into errors && ^
```

Comment:
I have never seen this construct before. Do you have some docs for this?

Reply:
which construct do you mean exactly? the `&&` does not let the next command execute if the preceeding command exits with non-0 exit code. the `^` is a line continuation in `cmd.exe`/`batch` (somehow it is not needed for `bash` case).

Reply:
I get it . It's actually a bit more subtle than than the `^` escapes (Ignores) the next char after it. That is a new line in this case at the end of a line it does indeed have this effect.